src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleared = sourceString.split(';');
+
+  return cleared.reduce((accum, style) => {
+    const n = style
+      .split(':')
+
+      .map((element) => element.trim());
+
+    const [key, value] = n;
+
+    accum[key] = value;
```

Comment:
[CODE PRACTICE] - Directly assigning properties to the accumulator without checking if the key is non-empty may result in undesired properties with 'undefined' values. Consider filtering out the empty keys before the assignment.